Week6/MyFavoriteMovies/app/src/main/java/com/raywenderlich/myfavoritemovies/repository/MovieRepository.kt

```diff
@@ -1,23 +1,20 @@
-package com.raywenderlich.myfavoritemovies.model
+package com.raywenderlich.myfavoritemovies.repository
 
 import com.raywenderlich.myfavoritemovies.App
+import com.raywenderlich.myfavoritemovies.model.Movie
 
 class MovieRepository {
 
-    private val dao = App.movieDb.movieDao()
+    fun deleteMovieById(movieId: Int) = dao.deleteMovieById(movieId)
 
     fun getAllMovies(): List<Movie> = dao.getAllMovies()
 
     fun getMovieById(movieId: Int?) = dao.getMovieById(movieId)
 
-    private fun saveMovies(movies: List<Movie>) = dao.insertMovies(movies)
-
     fun storeMoviesIfNotEmpty(movies: List<Movie>) {
         if (getAllMovies().count() < 1) saveMovies(movies)
     }
 
-//    fun addMovie(movie: Movie)
-//    fun getAllMovies(): LiveData<List<Movie>>
-//    fun deleteMovies(movie: Movie)
-//    fun deleteAllMovies()
-}
\ No newline at end of file
+    private val dao = App.movieDb.movieDao()
```

Comment:
Pass this in through the constructor, also would've loved an interface abstraction! And add suspend functions to Room operations!